guides/common/modules/con_types-of-provisioning-templates.adoc

```diff
@@ -17,13 +17,17 @@ For UEFI provisioning, select *PXEGrub2*.
 
 Finish::
 Post-configuration scripts to execute using an SSH connection when the main provisioning process completes.
-You can use Finishing templates only for imaged-based provisioning in virtual or cloud environments that do not support user_data.
+You can use Finish templates only for image-based provisioning in virtual or cloud environments that do not support user_data.
 Do not confuse an image with a foreman discovery ISO, which is sometimes called a Foreman discovery image.
 An image in this context is an install image in a virtualized environment for easy deployment.
 +
 When a finish script successfully exits with the return code `0`, {ProjectName} treats the code as a success and the host exits the build mode.
++
 Note that there are a few finish scripts with a build mode that uses a _call back_ HTTP call.
 These scripts are not used for image-based provisioning, but for post configuration of operating-system installations such as Debian, Ubuntu, and BSD.
+ifdef::satellite[]
+However, provisioning operating systems other than {RHEL} is not supported by {Team}.
```

Comment:
```suggestion
However, provisioning operating systems other than {RHEL} are not supported by {Team}.
```

Reply:
Actually, I mean `provisioning ... is not supported`, not `... systems are not supported`.
Perhaps "provisioning **of** operating systems ... is not supported"?
@bangelic